website/docs/r/nsxt_edgegateway_bgp_configuration.html.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_bgp_configuration"
+sidebar_current: "docs-vcd-resource-nsxt-edgegateway-bgp-configuration"
+description: |-
+  Provides a resource to manage BGP configuration on NSX-T Edge Gateway that has a dedicated Tier-0 
+  Gateway or VRF.
+---
+
+# vcd\_nsxt\_edgegateway\_bgp\_configuration
+
+Provides a resource to manage BGP configuration on NSX-T Edge Gateway that has a dedicated Tier-0
+Gateway or VRF. BGP makes core routing decisions by using a table of IP networks, or prefixes, which
+designate multiple routes between autonomous systems (AS).
+
+~> Only `System Administrator` can create this resource.
+
+~> In an NSX-T Edge Gateway that is connected to an external network backed by a VRF gateway, the
+local AS number and graceful restart settings are **read only**. Your system administrator can edit
+these settings on the parent Tier-0 gateway in NSX-T Data Center. 
+
+## Example Usage (Using BGP configuration for a dedicated Tier-0 gateway backed Edge Gateway)
+
+```hcl
+resource "vcd_nsxt_edgegateway_bgp_configuration" "testing" {
+  org = "my-org"
```

Comment:
missing #optional, in other examples too

Reply:
added everywhere, but missed to comment here.